lib/Resource/BaseWorkOSResource.php

```diff
@@ -61,7 +61,7 @@ public function __set($key, $value)
         }
 
         $msg = "${key} does not exist on " . static::class;
-        throw new UnexpectedValueException($msg);
+        throw new \WorkOS\Exception\UnexpectedValueException($msg);
```

Comment:
Needed to fully qualify the path to `UnexpectedValueException`, otherwise we'd get a different error when trying to throw it.